File: 0217-contains-duplicate/0217-contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
//         Arrays.sort(nums);
//         int m = nums.length;
//         for(int i = 1; i < m ; i++){
//             if(nums[i-1] == nums[i]){
//                 return true;
//             }
            
//         }
//         return false;
        
        
        HashSet<Integer> myArr = new HashSet<>();
        for (int num: nums){
           if(myArr.contains(num))
               return true;
            myArr.add(num);
        }
        return false;
        
    }
}

File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        # mydict = {}
        # for i in nums:
        #     if i not in mydict:
        #         mydict[i] = 0
        #     mydict[i] += 1
        
        # for value in mydict.values():
        #     if value > 1:
        #         return True
        # return False
        return len(set(nums)) != len(nums)

